sdk/core/core-tracing/README.md

```diff
@@ -16,40 +16,58 @@ npm install @azure/core-tracing
 
-## Examples
+### Span Propagation
 
-### Example 1 - Setting an OpenTelemetry Tracer
+Core Tracing supports both automatic and manual span propagation. Automatic propagation is handled using OpenTelemetry's API and will work well in most scenarios when run in `Node.js`.
 
-```js
-const opentelemetry = require("@opentelemetry/api");
-const { BasicTracer, SimpleSpanProcessor } = require("@opentelemetry/tracing");
-const { ZipkinExporter } = require("@opentelemetry/exporter-zipkin");
-const { setTracer } = require("@azure/core-tracing");
+For customers who require manual propagation, or to provide context propagation in the browser, all client library operations accept a `tracingContext` option under `tracingOptions` which allows you to manually pass the current context to the Azure SDK client library.
 
-const exporter = new ZipkinExporter({
-  serviceName: "azure-tracing-sample"
-});
-const tracer = new BasicTracer();
-tracer.addSpanProcessor(new SimpleSpanProcessor(exporter));
+### OpenTelemetry Compatibility
+
+Both the Azure SDK and Microsoft's [Application Insights](https://www.npmjs.com/package/applicationinsights) use [OpenTelemetry](https://opentelemetry.io/) to support tracing. Specifically, we depend on the [@opentelemetry/api](https://www.npmjs.com/package/@opentelemetry/api) npm package.
+
+As OpenTelemetry iterated on their API towards their 1.0 GA release, both libraries were updated to match.
 
-setTracer(tracer);
+Some incompatibility between the two libraries is due to mismatches between the OpenTelemetry versions used in either `@azure/core-tracing` or `applicationinsights` when the two are used side-by-side. For folks who are using both libraries in the same application, we recommend using the same version of OpenTelemetry for both libraries by upgrading to their latest versions.
 
-const rootSpan = tracer.startSpan("root");
-const context = opentelemetry.setSpan(opentelemetry.context.active(), rootSpan);
+> Please note that we do not foresee any future compatibility concerns now that OpenTelemetry 1.0.0 has been released and the API considered stable.
+
+#### Compatibility Matrix
+
+| Core Tracing     | Application Insights | @opentelemetry/api |
+| ---------------- | -------------------- | ------------------ |
+| 1.0.0-preview.10 |                      | 0.10.2             |
+| 1.0.0-preview.11 | 2.1.0                | 1.0.0-rc.0         |
+| 1.0.0-preview.11 | 2.1.1                | 1.0.0-rc.0         |
+| 1.0.0-preview.11 | 2.1.2                | 1.0.0-rc.0         |
+| 1.0.0-preview.11 | 2.1.3                | 1.0.0-rc.0         |
+| 1.0.0-preview.12 | ^2.1.4               | ^1.0.0             |
+| 1.0.0-preview.13 | ^2.1.4               | ^1.0.0             |
+
+Please see the [troubleshooting](#troubleshooting) section for additional information about handling compatibility errors.
+
+## Examples
```

Comment:
Should we add an example of how to use App Insights with Azure SDK?

Reply:
it's really orthogonal to the azure SDK - it would just be something like 

```
let appInsights = require("applicationinsights");
appInsights.setup("_your_ikey_").start();
```

Which doesn't feel very helpful here IMO - but I can be convinced otherwise...

Reply:
I was thinking along the same lines as https://github.com/Azure/azure-sdk-for-js/issues/15420
If we have deemed it necessary to call out how to use App Insights with Azure SDKs in the core/readme.md file, then should we do something similar in the readme for core-tracing as well.